libnavigation-base/src/test/java/com/mapbox/navigation/base/internal/route/NavigationRouteExTest.kt

```diff
@@ -0,0 +1,200 @@
+package com.mapbox.navigation.base.internal.route
+
+import com.mapbox.api.directions.v5.models.DirectionsResponse
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.api.directions.v5.models.Incident
+import com.mapbox.api.directions.v5.models.LegAnnotation
+import com.mapbox.api.directions.v5.models.RouteLeg
+import com.mapbox.navigation.base.route.NavigationRoute
+import com.mapbox.navigator.RouterOrigin
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class NavigationRouteExTest {
+
+    @Test
+    fun `extension NavigationRoute refreshRoute`() {
```

Comment:
No actions required, just a feedback from me. I find this "parametrised" tests hard to read. I spent 20 minutes and didn't really get all the scenarios. Usually if you write a test per scenario, tests are getting smaller and easier to understand

Reply:
thanks for the feedback, parametrized tests are helpful for different cases in the same scenario (like here), not setting up an environment for each test. I'd add more docs for current tests, to make it readable